src/System.Private.Uri/src/System/UriBuilder.cs

```diff
@@ -114,19 +81,12 @@ string extraValue
             {
                 Extra = extraValue;
             }
-            catch (Exception exception)
+            catch (ArgumentException)
```

Comment:
Why risk it?  What value does this change provide that makes it worthwhile, other than boosting code coverage numbers?